### PR TITLE
Added a matrix of deployment platforms and config files

### PR DESCRIPTION
**Pull request by sarahmaddox**
Fixes https://github.com/kubeflow/website/issues/1338

---
This change is [Reviewable](https://reviewable.io/reviews/kubeflow/website/1467)

### CODE REVIEW

**Comment by sarahmaddox** [comment]
This PR is now ready for review. Applying a hold to allow for multiple reviewers.
/hold

**Comment by sarahmaddox** [comment]
Preview: https://deploy-preview-1467--competent-brattain-de2d6d.netlify.com/docs/started/getting-started/#configuration-quick-reference

/assign @abhi-g

File: content/docs/started/getting-started.md [hunk 1]
```diff
@@ -46,6 +45,78 @@ to suit your environment (desktop or server, existing Kubernetes cluster or publ
...
+    <thead class="thead-light">
+      <tr>
+        <th>Deployment platform</th>
+        <th>Manifest</th>
+        <th>Deployment guide</th>
+      </tr>
+    </thead>
+    <tbody>
+      <tr>
+        <td><b>Existing Kubernetes cluster</b>: vanilla Kubeflow with no external
```
Comment (aarondietz234) [inline content/docs/started/getting-started.md]: Nit: I don't mind "vanilla" at all but if a word like "basic" or "standard" would still be accurate, that may be better for people less familiar with English.

**Comment by abhi-g** [comment]
Thanks @sarahmaddox. Overall looks good to me. Is there someone from Azure and IBM Cloud who can confirm their config files to use?
Rows 1 and 3 in the table have colons whereas others don't. 
Also Row 1 for Existing K8s cluster configuration kfctl_k8s_istio.0.7.0.yaml does install istio as well.

File: content/docs/started/getting-started.md (hunk 1 above)
Comment (sarahmaddox) [reply content/docs/started/getting-started.md]: Thanks! Done.

**Comment by sarahmaddox** [comment]
Thanks @abhi-g! I've updated the doc in response to your comments. 

@vjrantal Can you take a look at the config file I've listed for Microsoft Azure? I chose the config file that's used as the example in this doc: https://www.kubeflow.org/docs/azure/deploy/install-kubeflow/

@jinchihe Can you take a look at the config file I've listed for IBM Cloud Private? I chose the config file that's used in this doc: https://www.kubeflow.org/docs/started/cloud/getting-started-icp/

/cc @vjrantal
/cc @jinchihe

Preview: https://deploy-preview-1467--competent-brattain-de2d6d.netlify.com/docs/started/getting-started/#configuration-quick-reference

**Comment by jinchihe** [comment]
@sarahmaddox Great thanks!

/lgtm

**Comment by sarahmaddox** [comment]
/hold cancel

@aronchick please would you take a look at this PR on behalf of Microsoft? I think @vjrantal must be away, and I don't have any other contacts.

**Comment by sarahmaddox** [comment]
@abhi-g I'm having trouble finding someone to check the for Microsoft Azure side of things. The matrix shows the config file that's used as the example in this doc: https://www.kubeflow.org/docs/azure/deploy/install-kubeflow/

I could add the words "as an example" in the matrix too, or we can remove the config file from the matrix and just refer to the doc, or was can leave the matrix as is. What do you think?

**Comment by abhi-g** [comment]
I'll say we have it a fair try. It's goog to merge as it is. If they find someone who can correct it, they can always submit a PR to update.

/approve

**Pull request merged**